.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+    csslint:
+        ignore: 'box-model,ids,order-alphabetical'
```

Comment:
[REQUIRED]
You should use this configuration instead
![Screenshot from 2019-08-21 07-45-20](https://user-images.githubusercontent.com/30977845/63425794-bf8c1080-c3e7-11e9-8561-ce21792cd80d.png)
Don't forget to put the stylelint.config.js file in the root of your project
Check out this [link](https://github.com/microverseinc/linters-config/tree/master/css).